docs/modules/plugins/partials/generic-ui-steps.adoc

```diff
@@ -29,7 +29,7 @@ When I wait until element located by `name(welcome-image)` appears
 Waits for disappearance of the element by the locator.
 
 NOTE: If the element doesn't exist on the page/context, the step will immediately complete successfully.
-Checking the element on the page (if needed) should be done in a separate step (e.g. <<_wait_for_element_appearance>> or xref:plugin-html.adoc#_validate_elements[Validate elements]).
+Checking the element on the page (if needed) should be done in a separate step (e.g. <<_wait_for_element_appearance>> or xref:plugins:plugin-html.adoc#_validate_elements[Validate elements]).
```

Comment:
why is this change needed?

Reply:
I wouldn't say it's crucial, but detaching just specific steps will complicate overall structure and maintenance in my opinion. This tutorial isn't considered as the first step in learning course and thus might contain some "specific" info.